knowledge/store.py

```
@@ -0,0 +1,483 @@
+                        "INSERT INTO chunks_fts(chunks_fts) VALUES('rebuild')"
+                    )
+                except sqlite3.OperationalError as exc:
+                    log.debug("[knowledge] FTS rebuild skipped: %s", exc)
+            else:
+                log.info(
+                    "[knowledge] FTS5 unavailable — search will use LIKE fallback"
+                )
+            db.commit()
+            db.close()
+        except sqlite3.OperationalError as exc:
+            log.error("[knowledge] schema init failed at %s: %s", self.path, exc)
```

Ruff (0.15.12)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)